src/facewidgetscene.cpp

```diff
@@ -1,4 +1,5 @@
 /*
```

Comment:
**Reporter**: sneha-nd
**Rule**: whitespace/braces
**Severity**: ERROR
**File**: src/facewidgetscene.cpp L33

Missing space before {

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-1058474823>*